index.js

```diff
@@ -3,7 +3,7 @@ import * as Sentry from '@sentry/node'
 import { preprocess } from './lib/preprocess.js'
 import { evaluate } from './lib/evaluate.js'
 import { RoundData } from './lib/round.js'
-import { updateTopMeasurementParticipants } from './lib/platform-stats.js'
+import { periodicDatabaseRefresh } from './lib/platform-stats.js'
```

Comment:
can we rename this function to `refreshDatabase`? It shouldn't be concerned with how it's being called (periodic or not)

Reply:
Addressed in https://github.com/filecoin-station/spark-evaluate/pull/311/commits/e3845c9bd36cfddb719459fe72c3f2389b94ecda

Reply:
you renamed it to `databaseRefresh`, can we do the more intuitive `refreshDatabase`? To me functions should be verbs, as they're doing something